diff -ruNp cumbia-qtcontrols.orig/cumbia-qtcontrols.pro cumbia-qtcontrols/cumbia-qtcontrols.pro
--- cumbia-qtcontrols.orig/cumbia-qtcontrols.pro	2023-03-01 15:39:27.802871790 +0000
+++ cumbia-qtcontrols/cumbia-qtcontrols.pro	2023-03-01 15:42:06.681944480 +0000
@@ -285,28 +285,6 @@ unix:LIBS -= \
     -l$${cumbia_qtcontrols_LIB} -lpthread
 
 unix {
-
-DOXYGEN_BIN = $$system(which doxygen)
-isEmpty(DOXYGEN_BIN) {
-    message("cumbia-qtcontrols.pro: doxygen not found")
-} else {
- #   message("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!")
-#    message("---------------------------- DOCS DISABLED ----------------------")
-    message("-")
-    message("-")
-    message("Doxygen found")
-
-    doc.commands = \
-    doxygen \
-    Doxyfile;
-
-    doc.files = doc/*
-    doc.path = $${CUMBIA_QTCONTROLS_DOCDIR}
-    QMAKE_EXTRA_TARGETS += doc
-    !android-g++ {
-        INSTALLS += doc
-    }
-}
     inc.files = $${HEADERS}
     inc.path = $${CUMBIA_QTCONTROLS_INCLUDES}
 
diff -ruNp cumbia-qtcontrols.orig/qml/cumbia-qtcontrols-qml.pro cumbia-qtcontrols/qml/cumbia-qtcontrols-qml.pro
--- cumbia-qtcontrols.orig/qml/cumbia-qtcontrols-qml.pro	2023-03-01 15:39:27.890871288 +0000
+++ cumbia-qtcontrols/qml/cumbia-qtcontrols-qml.pro	2023-03-01 15:40:59.402341991 +0000
@@ -152,15 +152,3 @@ unix:android-g++ {
 
     INSTALLS += target qmldir other_files designer designer_imgs
 }
-
-unix {
-
-    doc.commands = \
-    doxygen \
-    Doxyfile;
-
-    doc.files = doc/*
-    doc.path = $${CUMBIA_QTCONTROLS_QML_DOCDIR}
-    QMAKE_EXTRA_TARGETS += doc
-}
-
